ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,131 @@
+
+.. _beginner-cloud:
+
+Confluent Cloud CLI
+===================
+
+Overview
+--------
+
+You can use `Confluent Cloud CLI
+<https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+to interact with your `Confluent Cloud
+<https://confluent.cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+cluster.
+
+`start.sh <start.sh>`__ is a fully scripted demo that shows you how to interact
+with |ccloud|. Using the |ccloud| CLI, you can complete the following workflow
+in about 8 minutes:
+
+-  Create a new environment and specify it as the default
+-  Create a new |ak| cluster and specify it as the default
+-  Create a user key/secret pair and specify it as the default
+-  Produce and consume with |ccloud| CLI
+-  Create a service account key/secret pair
+-  Run a Java producer: before and after ACLs
+-  Run a Java producer: showcase a Prefix ACL
+-  Run |kconnect| and kafka-connect-datagen connector with permissions
+-  Run a Java consumer: showcase a Wildcard ACL
+-  Delete the API key, service account, |ak| topics, |ak| cluster, environment,
+   and the log files
+
+Prerequisites
+~~~~~~~~~~~~~~
+
+-  Access to a |ccloud| cluster
+-  Local install of `Confluent Cloud
+   CLI <https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+   v1.7.0 or later
+-  |ccloud| user credentials saved in ``~/.netrc`` (save with command ``ccloud login --save``)
+-  Docker and Docker Compose for the local |kconnect| worker
+-  ``timeout`` installed on your host
+-  ``mvn`` installed on your host
+-  ``jq`` installed on your host
+
+Confluent Cloud Promo Code
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first 20 users to sign up for `Confluent Cloud
+<https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+and use promo code ``C50INTEG`` will receive an additional $50 free usage
+(`details
+<https://www.confluent.io/confluent-cloud-promo-disclaimer/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__).
+
+
+Run the demo
+------------
+
+.. note::
+
+   This example uses real resources in |ccloud|, and it creates and deletes
+   topics, service accounts, API keys, and ACLs.
+
+To run this demo, execute the following command:
+
+.. code-block:: bash
+
+      ./start.sh
+
```

Comment:
Proposal: let's capture the output of a successful run and walk through it here.  (Let's sync on Slack)

Reply:
Got it